CONTRIBUTING.md

```diff
@@ -19,6 +19,7 @@ The following is a set of guidelines and helpful information for contributing to
 - [Additional Note](#additional-notes)
 
 ## Code of Conduct
+
```

Comment:
It would indeed be a good idea to put this in the style guide. I put this in for compatibility because some markdown flavors require a blank line above and below in the case of ATX headings (some others require only one below).

Speaking of ATX headings, it would also probably be a good idea to mention the recommendation of only using non-closed ATX headings, and not Setext headings and/or closed ATX headings. See [this](https://webcoder.info/markdown-headers.html) for reference.